### DISCUSSION TITLE
fn keyword

### DISCUSSION THREAD

**Discussion by rraammiinn** [linkdd/letlang#4] [General]
I don't know anything about this language but if it turns out a great language func would be so annoying .
I think fn is more beautiful .

**Comment by linkdd** [comment]
> if it turns out a great language

I wish 😅

> func would be so annoying. I think fn is more beautiful.

This is just a matter of taste. Some people prefer `function`, some people would argue that functions must be pure and this would be "routines" and would prefer `def` (like Python) or `sub` (like Perl).

I settled on `func` (like Go) and I do like it this way.

I prefer focusing on semantics and the implementation and leave that kind of details for later.

> I don't know anything about this language

Feel free to consult the website and the source code 🙂

**Reply by rraammiinn** [reply]
ok but beauty matters .

**Reply by linkdd** [reply]
Beauty is subjective.

This discussion is not constructive, I'm closing it.

**Reply by athas** [reply]
I think `𝒻𝓊𝓃𝒸` is even more beautiful.
